range unavailable

iskettaneh opened the pull request:
Just a prototype

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/140006)
<!-- Reviewable:end -->

iskettaneh committed:
kvserver: increase LossQuorumCauseLeaderWatcherToSignalUnavailable verbosity

This commit increases the verbosity of the test:
TestLossQuorumCauseLeaderWatcherToSignalUnavailable to help make debugging
failures easier.

References: #143207

Release note: None
